man/nss-resolve.xml

```diff
@@ -141,6 +141,17 @@
         <xi:include href="version-info.xml" xpointer="v250"/></listitem>
       </varlistentry>
     </variablelist>
+
+    <variablelist class='environment-variables'>
+      <varlistentry>
+        <term><varname>$SYSTEMD_NSS_RESOLVE_INTERFACE</varname></term>
+
+        <listitem><para>Takes an interface name or index as an argument. When specified and non-zero, answers
```

Comment:
`and non-zero` sounds spurious... Please drop.

Reply:
Resolved in latest push.